### PR TITLE
Churn rate

### PR DESCRIPTION
**Pull request by AleckVictor**
Creating a separate model for churn rate.

### CODE REVIEW

File: models/mart/churn.sql [hunk 1]
```diff
@@ -0,0 +1,9 @@
+select 
+    date_trunc('month', mrr_month )::date as churn_month, 
+    customer_id,
+    name,
+    stripe_account 
+from {{ref('mrr_movements_monthly')}} mmm 
+where event_type = 'Churn'
+    and customer_id NOT IN ('cus_MVjwgFklliUF9p','cus_J8IS1IGMxzZLzR')
```
Comment (fmluizao) [inline models/mart/churn.sql:8]: Would be nice to add a comment to explain why we are excluding these customers.

File: models/mart/churn_rate.sql [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+    order by 1
+),
+active_customers as (
+	select 
+        date_trunc('month', "date")::date as m_month, 
+        count(distinct customer_id) as customer_count, 
+        stripe_account 
+	from {{ref('historical_mrr')}}  hm 
+	where mrr <> 0 
+		and customer_id NOT IN ('cus_MVjwgFklliUF9p','cus_J8IS1IGMxzZLzR')
```
Comment (fmluizao) [inline models/mart/churn_rate.sql:17]: ditto

File: models/mart/churn_rate.sql [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+),
+general_churn as (
+	select 
+		churn_month,
+		sum(customer_count) as churn_customers,
+        stripe_account
+	from churn_count
+	group by 1,3
+	order by 1	
+		),
```
Comment (fmluizao) [inline models/mart/churn_rate.sql:29]:
```suggestion
),
```

File: models/mart/churn_rate.sql [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+general_calc as (
+	select 
+        m_month,
+		sum(c.churn_customers) / sum(customer_count) as churn_rate
+	from active_customers ac
+		left join general_churn c on c.churn_month = ac.m_month
+    group by 1),
```
Comment (fmluizao) [inline models/mart/churn_rate.sql:36]:
```suggestion
    group by 1
),
```

File: models/mart/churn_rate.sql [hunk 5]
```diff
@@ -0,0 +1,70 @@
...
+us_calc as (
+	select 
+        m_month,
+		sum(c.churn_customers) / sum(customer_count) as churn_rate
+	from active_customers ac
+		left join general_churn c on c.churn_month = ac.m_month and ac.stripe_account = c.stripe_account
+	where ac.stripe_account = 'us'
+    group by 1),
```
Comment (fmluizao) [inline models/mart/churn_rate.sql:44]:
```suggestion
    group by 1
),
```

File: models/mart/churn_rate.sql [hunk 6]
```diff
@@ -0,0 +1,70 @@
...
+br_calc as (
+	select 
+        m_month,
+		sum(c.churn_customers) / sum(customer_count) as churn_rate
+	from active_customers ac
+		left join general_churn c on c.churn_month = ac.m_month and ac.stripe_account = c.stripe_account
+	where ac.stripe_account = 'br'
+    group by 1)
```
Comment (fmluizao) [inline models/mart/churn_rate.sql:52]:
```suggestion
    group by 1
)

```

**Review by fmluizao — APPROVED** [review]
LGTM :+1:

File: models/mart/churn.sql (hunk 1 above)
Comment (AleckVictor) [reply models/mart/churn.sql:9]: Shure! Thanks for noticing 👍

**Pull request merged**